microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -609,3 +609,11 @@ def verify_uio_binding(
     def _force_dpdk_default_source(self, variables: Dict[str, Any]) -> None:
         if not variables.get("dpdk_source", None):
             variables["dpdk_source"] = "https://dpdk.org/git/dpdk-stable"
+
+    def after_case(self, log: Logger, **kwargs: Any) -> None:
```

Comment:
Would we be able to get away with just a reboot? I'm not sure if the modprobe and driver binds persist, since we don't know the state on failure it might be easier to just reset.

Reply:
reboot also works.

Reply:
reboot takes longer time. But if some distros require reboot, we have to add it.

Reply:
I'd like to merge it now.